apps/sensenet/src/components/settings/stats.tsx

```diff
@@ -0,0 +1,45 @@
+import { useRepository, VersionInfo } from '@sensenet/hooks-react'
+import { Container } from '@material-ui/core'
+import clsx from 'clsx'
+import React, { useEffect, useState } from 'react'
+import { useGlobalStyles } from '../../globalStyles'
+import { useLocalization } from '../../hooks'
+import { ComponentsWidget } from './stats-components-widget'
+import { InstalledPackagesWidget } from './stats-installed-packages-widget'
+import { StorageWidget } from './stats-storage-widget'
+
+export const Stats: React.FunctionComponent = () => {
+  const globalClasses = useGlobalStyles()
+  const localization = useLocalization()
+  const repository = useRepository()
+  const [data, setData] = useState<VersionInfo>()
+
+  useEffect(() => {
+    ;(async () => {
+      const response = await repository.executeAction<any, VersionInfo>({
+        idOrPath: '/Root',
+        name: 'GetVersionInfo',
+        method: 'GET',
+      })
+
+      setData(response)
+    })()
+  }, [repository])
+
+  if (!data) return null
```

Comment:
GetVersionInfo and GetDashboardData requests are nonparallel this way, causing longer data loading.

All the data loading should happen here with a loading indicator and render the whole page at once.